source/install/cloud-changelog.md

```diff
@@ -4,12 +4,49 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2022-06-29](#release-2022-06-29)
 - [Release 2022-06-22](#release-2022-06-22)
 - [Release 2022-06-15](#release-2022-06-15)
 - [Release 2022-06-14](#release-2022-06-14)
 - [Release 2022-05-26](#release-2022-05-26)
 - [Release 2022-05-12](#release-2022-05-12)
-- [Release 2022-04-28](#release-2022-04-28)
+
+## Release 2022-06-29
+
+### Highlights
+
+#### Insights (Beta)
+ - Added workplace insights consisting of usage and behaviour data, which helps Enterprises further increase productivity of their employees through Mattermost functionality.
+
+### Improvements
+
+#### User Interface (UI)
+- Added support for syntax highlighting for 1C:Enterprise (BSL) language.
```

Comment:
I'm unfamiliar with this "language". Does it make sense to elaborate a bit on the value of this support for those who are unfamiliar with it?

Reply:
I'll add @hmhealey to confirm the benefit/value.

Reply:
I don't think we need to. I honestly had to google the language to find out what it was, particularly because a lot of the documentation was in a Cyrillic language that I couldn't read. It's supported by the syntax highlighting library we use though, so it was basically free to include